lib/postgrex/extensions/bit_string.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Postgrex.Extensions.BitString do
+  use Postgrex.BinaryExtension,
+    [receive: "bit_recv", send: "bit_send", type: "bit",
+     receive: "varbit_recv", send: "varbit_send", type: "varbit"]
```

Comment:
We just use `send` entries because that describes the wire encoding. Using `type` can lead to issues with custom types. Postgrex can't handle the situation where a `send` and `receive` use non-matching encoding so we skip `receive`.